src/engine/rendertext.cpp

```diff
@@ -712,26 +714,25 @@ float key_widthf(const char *str)
 {
     const char *keyn = str;
     if(*str == '=') keyn = gettklp(++str);
-    vector<char *> list;
+    std::vector<std::string> list;
     explodelist(keyn, list);
     float width = 0, scale = curfont->maxh*curfont->scale/float(curfont->defaulth)*curtextscale*textkeyimagescale;
     loopv(list)
     {
         if(i && textkeyseps) width += text_widthf("|");
         if(textkeyimages)
         {
-            textkey *t = findtextkey(list[i]);
+            textkey *t = findtextkey(list[i].data());
```

Comment:
same here, if you are editing the line anyways, please change it to `textkey* t`